src/FEValues/interface_values.jl

```diff
@@ -0,0 +1,372 @@
+            f_value_here = $(f_)(iv, qp, u_a, dof_range_a, use_element_a = true)
+            f_value_there = $(f_)(iv, qp, u_b, dof_range_b, use_element_a = false)
+            return f_value_there - f_value_here 
+        end
+        # TODO: Deprecate this, nobody is using this in practice...
+        function $(func)(iv::InterfaceValues, qp::Int, u_a::AbstractVector{<:Vec}, u_b::AbstractVector{<:Vec})
+            f_value_here = $(f_)(iv, qp, u_a, use_element_a = true)
+            f_value_there = $(f_)(iv, qp, u_b, use_element_a = false)
+            return f_value_there - f_value_here 
+        end
+    end
+end
+
+"""
+    transform_interface_points!(dst::Vector{Vec{dim, Float64}}, iv::InterfaceValues, points::Vector{Vec{dim, Float64}}, grid::AbstractGrid, face_a::FaceIndex, face_b::FaceIndex)
+
+Transform the points from face A to face B using the orientation information of the interface and store it in the vecotr dst.
+For 3D, the faces are transformed to regular polygons such that the rotation angle is the shift in reference node index × 2π ÷ number of edges in face.
+If the face is flipped then the flipping is about the axis that perserves the position of the first node (which is the reference node after being rotated to be in the first position,
+it's rotated back in the opposite direction after flipping).
+Take for example the interface
+```
+        2           3
+        | \\         | \\
+        |  \\        |  \\
+y       | A \\       | B \\ 
+↑       |    \\      |    \\
+→  x    1-----3     1-----2  
+```
+Transforming A to a equilateral triangle and translating it such that {0,0} is equidistant to all nodes
+```
+        3
+        +
+       / \\         
+      /   \\
+     /  x  \\                          
+    /   ↑   \\                        
+   /  ←      \\                          
+  /  y        \\                            
+2+-------------+1                     
+```
+Rotating it -270° (or 120°) such that the reference node (the node with smallest index) is at index 1
+```
+        1
+        +
+       / \\         
+      /   \\
+     /  x  \\                          
+    /   ↑   \\                        
+   /  ←      \\                          
+  /  y        \\                            
+3+-------------+2                     
+```
+Flipping about the x axis (such that the position of the reference node doesn't change) and rotating 270° (or -120°)
+```
+        2
+        +
+       / \\         
+      /   \\
+     /  x  \\                          
+    /   ↑   \\                        
+   /  ←      \\                          
+  /  y        \\                            
+3+-------------+1                     
+```
+Transforming back to the reference trianle
+```
+       3           
+       | \\
+       |  \\
+y      |   \\ 
+↑      |    \\
+→ x    1-----2  
```

Comment:
So, this is your reference triangle and not the actual reference triangle in Ferrite, right?

Reply:
My bad, the text for this one should've been "Transforming back to triangle B" not reference triangle.
The reference triangle is the same as Ferrite's one, which we transform to temporarily when flipping.